src/webchat-ui/components/history/ChatScroller.tsx

```diff
@@ -154,13 +158,14 @@ const ScrollerContent = ({ children, scrolledToLastInput, setShouldScrollToLastI
 	return (
 		<div>
 			{children}
-			{!sticky && (
+			{!sticky && !scrollButtonDisabled && (
 				<ScrollButton
 					className="webchat-scroll-to-bottom-button"
 					onClick={scrollToBottom}
 					aria-label="Scroll to bottom"
 				>
-					↓
+					{/* // Keep arrow symbol in parentheses!*/}
+					{"↓"}
```

Comment:
This does not solve the issue. I think we need to replace the down arrow with an SVG icon or inject the HTML entity (&#8595;) using innerHTML. Anyway, it is out-of-scope for your ticket

Reply:
Yeap. Lets fix this in separate item then,